### PR TITLE
bpo-36503: remove references to 'aix3' and 'aix4'

### PR DESCRIPTION
**Pull request by aixtools**
https://bugs.python.org/issue36503

### CODE REVIEW

**Comment by ZackerySpytz** [comment]
Why are there two news entries?

File: Lib/test/test_fcntl.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def get_lockdata():
                                fcntl.F_WRLCK, 0)
     elif sys.platform.startswith('gnukfreebsd'):
         lockdata = struct.pack('qqihhi', 0, 0, 0, fcntl.F_WRLCK, 0, 0)
-    elif sys.platform in ['aix3', 'aix4', 'hp-uxB', 'unixware7']:
+    elif sys.platform in ['hp-uxB', 'unixware7']:
```
Comment (serhiy-storchaka) [inline Lib/test/test_fcntl.py:37]: Should not 'aix5', 'aix6', etc be added instead?

File: Misc/NEWS.d/next/Tests/2019-04-02-09-25-31.bpo-36503.KrA0WF.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Remove references to "aix3" and "aix4"
```
Comment (serhiy-storchaka) [inline Misc/NEWS.d/next/Tests/2019-04-02-09-25-31.bpo-36503.KrA0WF.rst]:
```suggestion
Remove references to "aix3" and "aix4".
```

File: Misc/NEWS.d/next/Tests/2019-04-02-09-25-31.bpo-36503.KrA0WF.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+patch by M Felt
```
Comment (serhiy-storchaka) [inline Misc/NEWS.d/next/Tests/2019-04-02-09-25-31.bpo-36503.KrA0WF.rst]:
```suggestion
Patch by M. Felt.
```

File: Misc/NEWS.d/next/Build/2019-04-02-09-25-23.bpo-36503.0xzfkQ.rst [hunk 4]
```diff
@@ -0,0 +1,2 @@
+Remove references to "aix3" and "aix4"
```
Comment (serhiy-storchaka) [inline Misc/NEWS.d/next/Build/2019-04-02-09-25-23.bpo-36503.0xzfkQ.rst]:
```suggestion
Remove references to "aix3" and "aix4".
```

File: Misc/NEWS.d/next/Build/2019-04-02-09-25-23.bpo-36503.0xzfkQ.rst [hunk 5]
```diff
@@ -0,0 +1,2 @@
...
+patch by M Felt
```
Comment (serhiy-storchaka) [inline Misc/NEWS.d/next/Build/2019-04-02-09-25-23.bpo-36503.0xzfkQ.rst]:
```suggestion
Patch by M. Felt.
```

File: setup.py [hunk 6]
```diff
@@ -1884,9 +1884,6 @@ def detect_tkinter(self):
         libs.append('tk'+ version)
         libs.append('tcl'+ version)
 
-        if HOST_PLATFORM in ['aix3', 'aix4']:
```
Comment (serhiy-storchaka) [inline setup.py:1887]: Is not this needed on newer AIX versions?

File: Misc/NEWS.d/next/Tests/2019-04-02-09-25-31.bpo-36503.KrA0WF.rst (hunk 3 above)
Comment (methane) [reply Misc/NEWS.d/next/Tests/2019-04-02-09-25-31.bpo-36503.KrA0WF.rst]: @aixtools Please add your name in `Misc/ACKS` and use same notation in NEWS entry.

File: Lib/test/test_fcntl.py (hunk 1 above)
Comment (aixtools) [reply Lib/test/test_fcntl.py:37]: replied by email, but to be sure - adding here as well.

-- No --. This has never been an issue with any AIX version 5, 6 or 7. This is probably something that got corrected as IBM added "Linux Affinity" back in 1999-2000.

File: Misc/NEWS.d/next/Build/2019-04-02-09-25-23.bpo-36503.0xzfkQ.rst (hunk 4 above)
Comment (aixtools) [reply Misc/NEWS.d/next/Build/2019-04-02-09-25-23.bpo-36503.0xzfkQ.rst]: I guess I could have just pressed the button above. Next time :) - Changed, in any case.

File: setup.py (hunk 6 above)
Comment (aixtools) [reply setup.py:1887]: No. see earlier comment about my guess that this was fixed back when Linux Affinity was "added" to AIX.

**Review by methane — APPROVED** [review]

**Pull request merged**